src/i18n/locales/en/translation.json

```diff
@@ -258,10 +259,25 @@
         }
       },
       "btcpay": {
-        "title": "BTCPay Server",
+        "title": "BTCPay Server"
+      },
+      "btcpay_lnd": {
+        "page": {
+          "title": "Connect to BTCPay LND",
+          "instructions": "Navigate to your BTCPayServer and log in as an admin. Go to Server Settings > Services > LND REST - See information. Then Click \"See QR Code information\" and copy the QR Code data. Paste it below:"
+        },
+        "config": {
+          "label": "Config data",
+          "placeholder": "config=https://your-btc-pay.org/lnd-config/212121/lnd.co"
+        },
+        "errors": {
+          "connection_failed": "Connection failed. Is the BTCPay connection URL correct and accessible?"
+        }
+      },
+      "btcpay_commando": {
         "page": {
-          "title": "Connect to your BTCPay LND node",
-          "instructions": "Navigate to your BTCPayServer and log in as an admin. Go to Server Settings > Services > LND Rest - See information. Then Click \"See QR Code information\" and copy the QR Code data. Paste it below:"
+          "title": "Connect to BTCPay CLN",
```

Comment:
Use `Core Lightning` since that is the official product name.

Reply:
![image](https://github.com/getAlby/lightning-browser-extension/assets/55848322/8b37bf94-2f43-4851-811c-09278f704c13)
at start we use core lightning just to keep it shorter headings we used CLN there. maybe then we change CLN to core lightning everywhere to be consistent with

Reply:
I appreciate you implementing these changes, but they are completely unrelated to the new onboarding (and need to separately be tested, for that we need to setup a BTCPay instance with a Core Lightning node). 

To not block the onboarding related changes any further, please move all the BTCPayServer related changes to a dedicated PR so we can test / review that separately.

Reply:
resolved